apps/docs/src/styles/_code-snippet.scss

```
@@ -0,0 +1,6 @@
+@use '../dist/components/core/styles/common/vendor-prefixes';
```

мб так? нужно проверить, что это будет работать
```suggestion
@use '@koobiq/components/core/styles/common/vendor-prefixes';
```

---

он так не может резолвить , сделал так: `@use 'components/core/styles/common/vendor-prefixes';`